### PR TITLE
Skip some unsupported onnx backend tests

### PR DESCRIPTION
**Pull request by houseroad**
Later I will check more onnx backend tests to onnx repo.

@ezyang @bddppq @yinghai

### CODE REVIEW

File: caffe2/python/onnx/tests/onnx_backend_test.py [hunk 1]
```diff
@@ -22,6 +22,14 @@
                      '|test_mean|test_hardmax'  # Does not support Mean and Hardmax.
                      '|test_cast.*FLOAT16.*'  # Does not support Cast on Float16.
                      '|test_depthtospace.*'  # Does not support DepthToSpace.
+                     '|test_reduce_l1.*'  # Does not support ReduceL1.
+                     '|test_reduce_l2.*'  # Does not support ReduceL2.
+                     '|test_reduce_log_sum.*'  # Does not support ReduceLogSum.
+                     '|test_reduce_max.*'  # Does not support ReduceMax.
```
Comment (bddppq) [inline caffe2/python/onnx/tests/onnx_backend_test.py]: It's good to prioritize this to have better support of exporting `torch.max(tensor, dim=n)`

**Review by bddppq — APPROVED** [review]

File: caffe2/python/onnx/tests/onnx_backend_test.py (hunk 1 above)
Comment (houseroad) [reply caffe2/python/onnx/tests/onnx_backend_test.py]: @bddppq yeah, @BIT-silence is working on it right now, I think.

**Review by yinghai — COMMENTED** [review]
Why are they unsupported? No ops in c2?

**Review by yinghai — APPROVED** [review]
Synced offline. C2 doesn't have support for some of the ops yet.

**Pull request merged**